packages/runtime/src/lib/constants.ts

```diff
@@ -10,6 +10,7 @@ export const ALGORAND_ACCOUNT_MIN_BALANCE = 1e6; // 1 ALGO
 // values taken from: https://developer.algorand.org/docs/features/asc1/stateful/#minimum-balance-requirement-for-a-smart-contract
 // minimum balance costs (in microalgos) for ssc schema
 export const APPLICATION_BASE_FEE = 0.1e6; // base fee for creating or opt-in to application
+export const ASSET_CREATION_FEE = 0.1e6; // creation fee for asset
```

Comment:
Can we merge these constants with a single name? Maybe `BASE_FEE`, as it's value is also used in `app-opt-in` & `asset-opt-in`

Reply:
not sure, this gives more clarity

Reply:
ok